File: sale-service/src/main/java/com/gallery/saleservice/domain/SaleID.java
package com.gallery.saleservice.domain;

public class SaleID {
    private int id;

    public SaleID() {
        this.id = 0;
    }

    public SaleID(int id) {
        this.id = id;
    }

    public SaleID(SaleID other) {
        this.id = other.id;
    }

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }
}

File: app/SaleService/Domain/SaleID.cs
namespace SaleService.Domain
{
    public class SaleID
    {
        private int _saleId;

        public SaleID()
        {
            this._saleId = 0;
        }

        public SaleID(int saleId)
        {
            this._saleId = saleId;
        }

        public SaleID(SaleID saleId)
        {
            this._saleId = saleId._saleId;
        }

        public int Id
        {
            get { return this._saleId; }
            set { this._saleId = value; }
        }
    }
}